lib/internal/buffer.js

```diff
@@ -468,6 +538,68 @@ function readDoubleForwards(offset = 0) {
 }
 
 // Write integers.
+function writeBigU_Int64LE(buf, value, offset, min, max) {
+  // TODO: perform any conversions on value?
```

Comment:
`TODO(seishun)`

Reply:
I'll just remove the comment. We can add coercion later if there's a need for it.